modules/bmo-about-the-baremetalhost-resource.adoc

```diff
@@ -0,0 +1,270 @@
+// This is included in the following assemblies:
+//
+// post_installation_configuration/bare-metal-configuration.adoc
+
+:_content-type: REFERENCE
+[id="about-the-baremetalhost-resource_{context}"]
+= About the BareMetalHost resource
+
+Metal^3^ introduces the concept of the `BareMetalHost` resource, which defines a physical host and its properties. The `BareMetalHost` resource contains two sections:
+
+. The bare metal host specification.
+. The bare metal host's current status.
+
+[discrete]
+== The BareMetalHost specification
```

Comment:
We should review this with any changes made to 4.10. We also should include the  *preprovisioningimages* resource here but that is a different set of documentation.